templates/cloudflare-pages/remix.env.d.ts

```diff
@@ -1,3 +1,3 @@
 /// <reference types="@remix-run/dev" />
-/// <reference types="@remix-run/cloudflare-pages/globals" />
+/// <reference types="@remix-run/cloudflare-pages" />
```

Comment:
This should point to the runtime globals instead

```suggestion
/// <reference types="@remix-run/cloudflare/globals" />
```